src/common/common.gyp

```diff
@@ -0,0 +1,26 @@
+{
+  'includes':[
+    'common.gypi',
+  ],
+  'targets': [{
+    'target_name': 'tizen_common',
+    'type': 'static_library',
+    'sources': [
+      'extension.cc',
+      'extension.h',
+      'logger.cc',
+      'logger.h',
+      'picojson.h',
+      'scope_exit.h',
+      'utils.h',
+      'XW_Extension.h',
+      'XW_Extension_EntryPoints.h',
+      'XW_Extension_Permissions.h',
+      'XW_Extension_Runtime.h',
+      'XW_Extension_SyncMessage.h',
+    ],
```

Comment:
The whole change solution looks good to me. How about implicitly enable C++11 in cflags: add `-std=c++11` and remove `-std=c++0x` if exists.